_locales/fr/messages.json

```
@@ -1,279 +1,279 @@
 {
     "hash_status_code_1": {
         "message": "à jour",
-        "description": "Les règles ClearURLs sont parfaitement à jour."
+        "description": "Les règles ClearURLs sont à jour."
     },
     "hash_status_code_2": {
         "message": "mis à jour",
         "description": "Cela indique la réussite des mises à jour des règles de ClearURLs."
     },
     "hash_status_code_3": {
         "message": "mise à jour disponible",
-        "description": "Une mis à jour pour les règles de ClearURLs sont disponibles."
+        "description": "Une mis à jour pour les règles de ClearURLs est disponible."
     },
     "hash_status_code_4": {
         "message": "erreur",
-        "description": "ClearURLs n’a pas pu démarrer adéquatement pour son bon fonctionnement."
+        "description": "ClearURLs n’a pas pu démarrer correctement."
     },
     "hash_status_code_5": {
-        "message": "Quelque chose a dysfonctionné",
-        "description": "Pendant la mis à jour des règles ClearURLs un dysfonctionnement est survenue."
+        "message": "Oh non! Un problème est survenu!",
+        "description": "Un problème est survenu pendant la mis à jour des règles ClearURLs"
     },
     "log_redirect": {
-        "message": "Cette URL a été redirigée",
-        "description": "Cette chaîne est utilisée pour les redirections dans le protocole ClearURL."
+        "message": "Cette URL a été redirigé",
```

```suggestion
        "message": "Cette URL a été redirigée",
```
while both "cet" and "cette" are correct, the last words needs to be consistent with the gender of "cet"